lib/stack_master/validator.rb

```diff
@@ -10,12 +10,12 @@ def initialize(stack_definition, config)
     end
 
     def perform
-      parameter_hash = ParameterLoader.load(@stack_definition.parameter_files)
-      compile_time_parameters = ParameterResolver.resolve(@config, @stack_definition, parameter_hash[:compile_time_parameters])
-
       StackMaster.stdout.print "#{@stack_definition.stack_name}: "
-      template_body = TemplateCompiler.compile(@config, @stack_definition.compiler, @stack_definition.template_dir, @stack_definition.template, compile_time_parameters, @stack_definition.compiler_options)
-      cf.validate_template(template_body: TemplateUtils.maybe_compressed_template_body(template_body))
```

Comment:
This template compilation logic is duplicated in the `Stack` class. It all seems to work by just using the `Stack` class directly via the refactoring in 6af37cbe81083480fafadac7bde75c51a83f0e64, and it keeps things DRY.